lib/Runtime/Language/InterpreterStackFrame.cpp

```diff
@@ -7935,39 +7935,12 @@ const byte * InterpreterStackFrame::OP_ProfiledLoopBodyStart(const byte * ip)
     template <class T>
     Var InterpreterStackFrame::OP_LdModuleSlot(Var instance, const unaligned T* playout)
     {
-        Js::SourceTextModuleRecord* moduleRecord = scriptContext->GetLibrary()->GetModuleRecord(playout->SlotIndex1);
-        Assert(moduleRecord != nullptr);
-
-        Var* moduleRecordSlots = moduleRecord->GetLocalExportSlots();
-        Assert(moduleRecordSlots != nullptr);
-
-        if (moduleRecord->GetLocalExportCount() <= (uint)playout->SlotIndex2)
-        {
-            Js::Throw::FatalInternalError();
-        }
-
-        Var value = moduleRecordSlots[playout->SlotIndex2];
-        Assert(value != nullptr);
-
-        return value;
+        return JavascriptOperators::OP_LdModuleSlot(playout->SlotIndex1, playout->SlotIndex2, scriptContext);
```

Comment:
It does check that both index values are correct in this path:

OP_LdModuleSlot->OP_GetModuleExportSlotAddress->OP_GetModuleExportSlotArrayAddress